insights/contrib/soscleaner.py

```diff
@@ -1014,8 +1014,8 @@ def _clean_line(self, line, filename):
                 if false_positive in filename:
                     process_obfuscation = False
             new_line = self._sub_keywords(line)  # Keyword Substitution
-            if self.obfuscate_macs is True:
-                new_line = self._sub_mac(new_line)  # MAC address obfuscation
+            # if self.obfuscate_macs is True:
```

Comment:
Committing commented out code is pointless. The commit tracks the changes; if the code ever needs to be restored, a revert or partial revert can restore the code.